frontend/src/views/ProfileView.vue

```diff
@@ -1,10 +1,74 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a profile page</h1>
+			<VerticalAvatarAndUserName
+				profile_picture="/test-profile.png"
+				:profile_name="this.user.name"
+			/>
+			<OverviewWithMidline :data-array="dataArray" />
 		</div>
 	</div>
 </template>
 
+<style scoped></style>
+
 <script lang="ts">
-</script>
\ No newline at end of file
+import { defineComponent } from 'vue';
+import VerticalAvatarAndUserName from '@/components/user-info/VerticalAvatarAndUserName.vue';
+import OverviewWithMidline from '@/components/overviews/OverviewWithMidline.vue';
+import { ref, onMounted, type Ref, defineProps } from 'vue';
+import type { OverviewArray } from '@/types/OverviewArray';
+import { getRequest } from '@/utils/apiRequests';
+import type { User } from '../types/User';
+import type { UserAchievement } from '../types/User';
+import type { Achievement } from '../types/User';
+
+let dataArray: Ref<OverviewArray[]> = ref([]);
+let user: User = {} as User;
+
+export default defineComponent({
+	name: 'ProfileView',
+	components: {
+		OverviewWithMidline,
+		VerticalAvatarAndUserName,
+	},
+	props: {
+		profile_id: String,
+	},
+	data() {
+		return {
+			user: {} as User,
+			userAchievement: [] as UserAchievement[],
+			achievement: [] as Achievement[],
+		};
+	},
+	async created() {
+		this.user = (await getRequest(`users/${this.profile_id}`)).data;
+		if (!this.profile_id) {
+			dataArray.value = [
+				{ left: 'this profile', right: 'does not exist' },
+			];
+		} else {
+			dataArray.value = [
+				{ left: 'intra name', right: this.user.name },
+				{ left: 'member since', right: this.user.created_at },
+				{ left: 'wins', right: 1 }, //needs to be connected
+				{ left: 'losses', right: 190 }, //needs to be connected
+				{
+					left: 'achievements',
+					right: this.user.achievements.map((ach) => {
+						return ach.name;
+					}),
```

Comment:
This makes the achievements into an array.. and displays it like this:

![Screen Shot 2023-02-13 at 3 21 53 PM](https://user-images.githubusercontent.com/1089484/218483486-c4bb2d16-d24d-43a0-8ec4-1bf4c21b403b.png)

So maybe for now add a join with a comma after the map? (`map().join(', ')`)